spec/jwk_spec.rb

```diff
@@ -75,7 +75,7 @@
       it 'is not ASN.1 encoded' do
         expect {
           OpenSSL::ASN1.decode(subject.sign(message))
-        }.to raise_error
+        }.to raise_error(OpenSSL::ASN1::ASN1Error)
```

Comment:
It's a bit of an unrelated change but being specific on the expected error does not hurt.